src/main/java/gyro/aws/acm/AcmCertificateFinder.java

```diff
@@ -119,12 +133,21 @@ protected List<CertificateDetail> findAws(AcmClient client, Map<String, String>
             filterBuilder = filterBuilder.keyUsageWithStrings(filters.get("key-algorithm"));
         }
 
+        List<String> certArns = new ArrayList<>();
```

Comment:
```suggestion
        Set<String> certArns = new LinkedHashSet<>();
```
Should this collection of ARNs be a `Set` to prevent duplicate results? Using a `LinkedHashSet` will preserve iteration order.